src/main/java/core/basesyntax/strategy/operation/BalanceImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.models.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+
+public class BalanceImpl implements OperationHandler {
```

Comment:
```suggestion
public class BalanceHandlerImpl implements OperationHandler {
```
change other handlers accordingly